apply/apply.go

```diff
@@ -180,12 +180,21 @@ func applyTree(dest afero.Fs, source *packr.Box, targetBasePath string, subst in
 			if e != nil {
 				return errors.Wrapf(e, "unable to create file %s", target)
 			}
+		} else if extension == ".rm" {
+			_, e = dest.Stat(target)
```

Comment:
could also always os.remove and check the error type

Reply:
Oh yeah I forgot there is a "file doesn't exist" error type.